clang/unittests/Format/FormatTest.cpp

```diff
@@ -24206,10 +24206,11 @@ TEST_F(FormatTest, IgnorePPDefinitions) {
   Style.IndentPPDirectives = FormatStyle::PPDIS_None;
   verifyNoChange("#if A\n"
                  "#define A  a\n"
-                 "#endif",
+                 "#endif\n",
```

Comment:
We normally don't have a trailing line break in the tests. Why did you add it?

Reply:
no reason. removed the trailing line breaks now